app/controllers/friendships_controller.rb

```diff
@@ -32,7 +32,14 @@ def destroy
   def accept_request
     @friend = User.find(params[:friend])
     current_user.confirm_friend(@friend)
+    create_inverse_friendship(@friend)
```

Comment:
By focusing only in Friendship model, you could have something like
```ruby
def confirm_friend
  self.update_attributes(confirmed: true)
  Friendship.create!(friend_id: self.user_id,
                  user_id: self.friend_id,
                  confirmed: true)
end
```

And Is better to move it on the model instead of the controller

Reply:
Thanks for the suggestion, it's fixed now